project/forms.py

```diff
@@ -3,8 +3,7 @@
 
 
 class FilterProjectForm(forms.Form):
-    title = forms.CharField(max_length=100, required=False , label="Búsqueda por título")
-    country = forms.CharField(max_length=100, required=False , label="Búsqueda por país")
+    search = forms.CharField(max_length=100, required=False , label="Búsqueda de título o pais")
```

Comment:
I would put only search in label, because it is what the other members of the group have put. You leave what seems best to you.